js/content-slider.js

```diff
@@ -69,17 +70,10 @@ export default class ContentSlider {
       controlsRight.addEventListener("click", (e) => {
         e.preventDefault();
         contentSliderCardsList.scrollLeft += 500;
-
-        // contentSliderCardsList.scroll({
-        //   left: 600, // absolute value. Needs to be dynamically calclucated.
-        //   behavior: "smooth"
-        // })
-      })
-
-      contentSliderControls.addEventListener("wheel", (e) => {
-        e.preventDefault();
-        contentSliderCardsList.scrollLeft += e.deltaY * 2;
     });
+
+    } else {
+      console.error("No content slider found");
```

Comment:
Just for a touch of extra context, could we prefix this error with something like `[ContentSlider]`?